Intermittent test failure: V4 API

```sh
======================================================================
FAIL: test_list_view_serializes_experiments (experimenter.experiments.tests.api.v4.test_views.TestExperimentListView)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/app/experimenter/experiments/tests/api/v4/test_views.py", line 39, in test_list_view_serializes_experiments
    self.assertEqual(serialized_experiments, json_data)
AssertionError: [OrderedDict([('id', None), ('arguments', [2917 chars]e)])] != [{'id': None, 'arguments': {'slug': None, [2572 chars]rue}]
```

Just saw it again on circle:

```
=================================== FAILURES ===================================
_________ TestExperimentListView.test_list_view_serializes_experiments _________
[gw2] linux -- Python 3.8.2 /usr/local/bin/python

self = <experimenter.experiments.tests.api.v4.test_views.TestExperimentListView testMethod=test_list_view_serializes_experiments>

    def test_list_view_serializes_experiments(self):
        experiments = []
        user_email = "user@example.com"
    
        for i in range(3):
            experiment = ExperimentFactory.create_with_variants(
                type=ExperimentConstants.TYPE_RAPID,
                objectives="gotta go fast",
                audience="us_only",
                features=["pinned_tabs"],
            )
            experiments.append(experiment)
    
        response = self.client.get(
            reverse("experiment-rapid-recipe-list"),
            **{settings.OPENIDC_EMAIL_HEADER: user_email},
        )
        self.assertEqual(response.status_code, 200)
    
        json_data = json.loads(response.content)
    
        serialized_experiments = ExperimentRapidRecipeSerializer(
            Experiment.objects.get_prefetched(), many=True
        ).data
    
>       self.assertEqual(serialized_experiments, json_data)
E       AssertionError: [OrderedDict([('id', None), ('arguments', [3411 chars]")])] != [{'id': None, 'arguments': {'slug': None, [3060 chars]e'"}]

experimenter/experiments/tests/api/v4/test_views.py:39: AssertionError
```

--- a/app/experimenter/experiments/tests/api/v4/test_views.py
+++ b/app/experimenter/experiments/tests/api/v4/test_views.py
@@ -15,6 +15,7 @@ class TestExperimentListView(TestCase):
 
         for i in range(3):
             experiment = ExperimentFactory.create_with_variants(
+                status=Experiment.STATUS_DRAFT,
                 type=ExperimentConstants.TYPE_RAPID,
                 objectives="gotta go fast",
                 audience="us_only",
@@ -31,13 +32,18 @@ class TestExperimentListView(TestCase):
             Experiment.objects.get_prefetched(), many=True
         ).data
 
+        self.maxDiff = None
         self.assertEqual(serialized_experiments, json_data)
 
 
 class TestExperimentRapidRecipeView(TestCase):
     def test_get_rapid_experiment_recipe_returns_recipe_info_for_experiment(self):
-        experiment = ExperimentFactory.create(
-            type=ExperimentConstants.TYPE_RAPID, audience="us_only"
+        experiment = ExperimentFactory.create_with_variants(
+            status=Experiment.STATUS_DRAFT,
+            type=ExperimentConstants.TYPE_RAPID,
+            objectives="gotta go fast",
+            audience="us_only",
+            features=["pinned_tabs"],
         )
 
         response = self.client.get(
@@ -47,4 +53,6 @@ class TestExperimentRapidRecipeView(TestCase):
         self.assertEqual(response.status_code, 200)
         json_data = json.loads(response.content)
         serialized_experiment = ExperimentRapidRecipeSerializer(experiment).data
+
+        self.maxDiff = None
         self.assertEqual(serialized_experiment, json_data)

